extern/sector-storage/stores/local.go

```diff
@@ -155,6 +157,8 @@ func (p *path) stat(ls LocalStorage) (fsutil.FsStat, error) {
 		}
 	}
 
+	log.Infow("storage stat", "took", time.Now().Sub(start), "reservations", len(p.reservations))
```

Comment:
You might want to remove this one because with a lot of storages connected this blows up all the logs everywhere

Reply:
runPoStCycle failed: running window post failed

why？